03-attribute-operations.qmd

```diff
@@ -240,29 +257,28 @@ Other options for the `aggfunc` parameter in `.dissolve` [include](https://geopa
 * `'mean'`
 * `'median'`
 
-Additionally, we can pass a custom function.
+Additionally, we can pass custom functions.
 
-as a more complicated example, here is how we can calculate the summed population, summed area, and count of countries, per continent. We calculate the `area_km2`+`pop` and the `name_long` (renamed to `n`) columns in two separate expressions (since we are using two different functions), then join the results. Also note that only one of the expression is a spatial aggregation, since the continent geometries are identical there is no need to calculate them twice:
+As a more complex example, here is how we can calculate the total population, area, and count of countries, per continent:
 
 ```{python}
-world_agg3a = world[['continent', 'area_km2', 'pop', 'geometry']] \
-    .dissolve(by='continent', aggfunc='sum')
-world_agg3b = world[['continent', 'name_long']] \
-    .groupby('continent') \
-    .nunique() \
-    .reset_index() \
-    .rename(columns={'name_long': 'n'})
-world_agg3 = pd.merge(world_agg3a, world_agg3b, on='continent')
+world_agg3 = world.dissolve(
+    by='continent', aggfunc={
+         "name_long": "count",
+         "pop": "sum",
+         'area_km2': "sum"
+     }).rename(columns={'name_long': 'n'})
```

Comment:
`aggfunc` accepts more than one function? Is this a new addition? Great to know and makes the code much better!